test/vagrant-service-manager/service_base_test.rb

```diff
@@ -0,0 +1,66 @@
+require_relative '../test_helper'
+require 'vagrant-service-manager/config'
+require 'vagrant-service-manager/plugin_util'
+
+module VagrantPlugins
+  describe ServiceManager::ServiceBase do
+    let(:machine) { fake_machine }
+    let(:instance) { ServiceManager::ServiceBase.new(machine, machine.env) }
+
+    it 'should build the proxy options if only proxy url is given' do
+      machine.config.servicemanager.http_proxy = 'foo:8080'
+      options = instance.proxy_cmd_options
+      assert_match(/HTTP_PROXY='foo:8080'/, options)
+    end
+
+    it 'should build the proxy options if all proxy settings are given' do
+      machine.config.servicemanager.http_proxy = 'foo:8080'
+      machine.config.servicemanager.http_proxy_user = 'user'
+      machine.config.servicemanager.http_proxy_password = 'password'
+
+      options = instance.proxy_cmd_options
+      assert_match(/HTTP_PROXY='foo:8080'/, options)
+      assert_match(/HTTP_PROXY_USER='user'/, options)
+      assert_match(/HTTP_PROXY_PASSWORD='password'/, options)
+    end
+
+    it 'should fail building proxy options if proxy url is not given' do
```

Comment:
The test is ok, but I think the expectation is wrongly formulated. The building of the proxy option is not failing. Failing would imply a exception or something. Is the expectation not more "should return empty proxy options if proxy url is not given"